src/JabbR-Core/Commands/CommandManager.cs

```diff
@@ -205,7 +205,13 @@ public static IList<ICommand> GetCommands()
             //var catalog = new AssemblyCatalog(typeof(CommandManager).Assembly);
             //var compositionContainer = new CompositionContainer(catalog);
             //return compositionContainer.GetExportedValues<ICommand>().ToList();
-            return new List<ICommand>() { new JoinCommand(), new OpenCommand(), new CreateCommand(), new LeaveCommand() };
+            return new List<ICommand>() {
```

Comment:
@KSLHacks &| @heatherbshapiro I recall providing some code that queried the commands successfully or is this in another unmerged branch? If so, lets have James merge that in here to avoid the future conflict (plus gives another set of eyes in testing it)